flink-tests/src/test/java/org/apache/flink/test/checkpointing/ResumeCheckpointManuallyITCase.java

```diff
@@ -244,7 +298,8 @@ private RocksDBStateBackend createRocksDBStateBackend(
     private static void testExternalizedCheckpoints(
             File checkpointDir,
             String zooKeeperQuorum,
-            StateBackend backend,
+            StateBackend previousBackend,
+            StateBackend newBackend,
```

Comment:
I prefer to introduce three state-backend for different three runs with different configurations. And we can actually use the same configuration for `run1` and `run3`.